### PR TITLE
Release note for COPY FROM STDIN

### PR DESCRIPTION
**Pull request by asenac**

### CODE REVIEW

File: doc/user/content/release-notes.md [hunk 1]
```diff
@@ -64,6 +64,9 @@ Wrap your release notes at the 80 character mark.
 - Support [equality operators](/sql/functions/#boolean) on
   [array data](/sql/types/array).
 
+- Add the [`COPY FROM`](/sql/copy-from) statement, which allows popullating a
```
Comment (awang) [inline doc/user/content/release-notes.md:67]: s/popullating/populating

File: doc/user/content/release-notes.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ Wrap your release notes at the 80 character mark.
...
+  table via the special PostgreSQL [`COPY` protocol][pg-copy].
```
Comment (awang) [inline doc/user/content/release-notes.md:68]: nit: i think "special" is unnecessary

**Review by awang — APPROVED** [review]
lgtm!  you have a spelling error though

File: doc/user/content/release-notes.md (hunk 2 above)
Comment (asenac) [reply doc/user/content/release-notes.md:68]: I copied it from the COPY TO :laughing:

**Pull request merged**